pkg/crc/machine/start.go

```diff
@@ -43,13 +43,17 @@ import (
 
 const minimumMemoryForMonitoring = 14336
 
-func getCrcBundleInfo(bundleName, bundlePath string) (*bundle.CrcBundleInfo, error) {
+func getCrcBundleInfo(preset crcPreset.Preset, bundleName, bundlePath string) (*bundle.CrcBundleInfo, error) {
 	bundleInfo, err := bundle.Use(bundleName)
 	if err == nil {
 		logging.Infof("Loading bundle: %s...", bundleName)
 		return bundleInfo, nil
 	}
 	logging.Debugf("Failed to load bundle %s: %v", bundleName, err)
+	bundlePath, err = bundle.Download(preset, bundlePath)
```

Comment:
I am not sure if a download should be started from a `getBundleInfo` function. Sounds like a responsibility it shouldn't have. It fails, as there is no info available...

in that case download and try again... but perhaps the check of the bundle exists should be done before you check for the bundle info.

Reply:
@gbraad Our assumption is user run the `crc setup` and that will automatic download the bundle and use but in case user want to try out different bundle without setup since other per-requisite are already in place then we use this to provide similar experience. Validation happen on cmd/start.go side where we show the warning but once we are in the pkg level we don't validate again and try to use what user provided and extract if it is a path or download it.